packages/commons-test/src/setupTestContainersVitest.ts

```diff
@@ -116,6 +117,9 @@ export function setupTestContainersVitest(
       await postgresDB?.none("TRUNCATE TABLE catalog.events RESTART IDENTITY");
       await postgresDB?.none("TRUNCATE TABLE tenant.events RESTART IDENTITY");
       await postgresDB?.none("TRUNCATE TABLE purpose.events RESTART IDENTITY");
+      await postgresDB?.none(
+        'TRUNCATE TABLE "authorization".events RESTART IDENTITY'
```

Comment:
```suggestion
        'TRUNCATE TABLE "authorization".events RESTART IDENTITY'
```
Why is it enclosed by ""?

Reply:
Because authorization is a keyword of PostgresDB so you can't call the table like this.
Therefore it was decided to put it in quotation marks